packages/docs/src/components/code-block/code-block.tsx

```diff
@@ -25,9 +24,11 @@ export const CodeBlock = component$((props: CodeBlockProps) => {
         : undefined;
   }
 
+  const theme = window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches ? 'dark' : 'light';
```

Comment:
This will not work in SSR... wondering if we remove all this JS, and do the trick with CSS!

Reply:
Right, I'll create two separate css files - one for light mode and one for dark mode, then import them conditionally:
```css
@import "./code-block.light.css" screen;
@import "./code-block.dark.css" screen and (prefers-color-scheme: dark);
```
This should work